hologram_dashboard/extension/src/app/app.constants.ts

```diff
@@ -0,0 +1,16 @@
+export class AppSettings {
+  public static API_ENDPOINT='http://localhost:8000/';
+}
+
+// Interface for keeping track of whether a data set is available for a specific
+// date.
+export interface DateAvailability {
+  date: string;
+  isAvailable: boolean;
+}
+
+// Interface for keeping track of all the availability of a specific source.
+export interface HologramDataAvailability {
+  sourceType: string;
+  dateAvailabilityList: Array<DateAvailability>;
+}
```

Comment:
these are not actually constants though. If these are only used in the dashboard.component.ts, you could just leave it there.